app/src/components/Contexts/Web3Provider.js

```diff
@@ -10,22 +10,31 @@ const getProvider = () => {
   return new ethers.providers.Web3Provider(window.ethereum, 'any');
 }
 
+const useContract = (provider, contractKey) => {
+  return React.useState(new ethers.Contract(
+    Contracts.ACTIVE_CONTRACTS[contractKey].address,
+    Contracts.ACTIVE_CONTRACTS[contractKey].BrownieOutput.abi,
+    provider.getSigner()
+  ));
+}
+
 export const Web3Provider = (props) => {
   const [provider, setProvider] = React.useState(getProvider());
   const [connectedAccount, setConnectedAccount] = React.useState("");
   const [chainId, setChainId] = React.useState(0);
   const [wolvercoinBalance, setWolvercoinBalance] = React.useState(0);
-  const [nftContract, setNftContract] = React.useState(new ethers.Contract(
```

Comment:
The reason for this was to prevent errors on pageload.  Creating web3Provider is problematic and we need to verify that no errors get thrown here.

Reply:
The move to use a hook should not change the behavior of the provider—it's just a wrapper for useState at the moment